wildfly-microprofile-rest-archetype/src/main/resources/archetype-resources/src/main/java/SampleEndPoint.java

```diff
@@ -0,0 +1,16 @@
+package ${package};
+import java.util.Date;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/Sample")
+public class SampleEndPoint {
+
+    @GET
```

Comment:
Not sure about the other files but this seems to have a mixture of tabs and spaces.